examples/ContainerRegistry.Tests.ps1

```diff
@@ -0,0 +1,32 @@
+BeforeAll {
+  Import-Module "../BenchPress/Helpers/Azure/ContainerRegistry.psm1"
+  Import-Module "../BenchPress/Helpers/Azure/Bicep.psm1"
+}
+
+Describe 'Verify Container Registry Exists' {
```

Comment:
Thanks Engin for the PR.

Can we also add a spin up/tear down module test example?

Also for the existing tests, including comments before the test method that highlights the difference (fetching an acr object and using the exists method that returns a boolean.) 

Thanks!